website_apps_store/controllers/main.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 BizzAppDev
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import base64
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.website.models.website import slug
+from odoo.addons.website.controllers.main import QueryURL
+from odoo.addons.website_sale.controllers.main import WebsiteSale, TableCompute
+
+_logger = logging.getLogger(__name__)
+
+PPG = 20  # Products Per Page
+PPR = 4   # Products Per Row
+
+
+class WebsiteSaleCustom(WebsiteSale):
+
+    def _get_search_domain(self, search, category, attrib_values):
+        domain = request.website.sale_product_domain()
+        if search:
+            for srch in search.split(" "):
+                domain += [
```

Comment:
Isn't better to make this search directly on product.product?

Reply:
_get_search_domain method is from Odoo website_sale module we have just added our conditions for the attribute, author and summary.

Reply:
Ah, OK.